test/integration/websocket_integration_test.cc

```diff
@@ -117,7 +117,7 @@ ConfigHelper::HttpModifierFunction setRouteUsingWebsocket() {
 }
 
 void WebsocketIntegrationTest::initialize() {
-  if (upstreamProtocol() != Http::CodecType::HTTP1) {
+  if (upstreamProtocol() == Http::CodecType::HTTP2) {
```

Comment:
Do we not need to allow_connect for HTTP/3 as well?  It should only work if configured.

Reply:
That config knob exists for h2, but not h1 nor h3

Reply:
I think we need to add that knob as part of this PR then.